### PR TITLE
feat: create terms of use page

### LINKED ISSUE #582: Create terms of use page
[FRO-54](https://linear.app/fuel-network/issue/FRO-54/create-terms-of-use-page)

Add terms of use as a page inside the application on the SignUp flow. This page should be the first one on the Flow. The terms will be rendered inside the component. The content should be markdown or plain HTML, no react.

Design: https://www.figma.com/file/uEckQY4EaQ5rXwyISg6d1g/Fuel-Wallet?node-id=3127%3A16992&t=JIOcIEePnitWrYMS-4

### PR DESCRIPTION
**Pull request by tomiiide**
This PR creates the terms and agreement page.

Closes #582

Screenshot 2023-02-27 at 1 40 02 PM

### CODE REVIEW

**Review by luizstacio — CHANGES_REQUESTED** [review]
We should hold this PR until we update the ToS content

**Review by luizstacio — APPROVED** [review]

File: packages/app/src/systems/SignUp/pages/TermsOfUse/TermsOfUse.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    setHasAcceptedTerms(false);
+    navigate(Pages.signUpWelcome());
+  };
+
+  return (
+    <Layout title="Terms of Service" isPublic>
+      <Stack gap="$8" align="center">
+        <Header
+          title="Terms of Service"
+          subtitle="Read and accept our terms of service"
```
Comment (LuizAsFight) [inline packages/app/src/systems/SignUp/pages/TermsOfUse/TermsOfUse.tsx:35]:
```suggestion
          subtitle="Check if you accept our terms of service"
```

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

**Comment by LuizAsFight** [comment]
e2e tests are broken

File: packages/app/src/systems/SignUp/pages/TermsOfUse/data/terms.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
+# Fuel Wallet Testnet Terms Of Use Agreement
```
Comment (luizstacio) [inline packages/app/src/systems/SignUp/pages/TermsOfUse/data/terms.md:1]:
```suggestion
## Fuel Wallet Testnet Terms Of Use Agreement
```

File: packages/app/src/systems/SignUp/pages/TermsOfUse/TermsOfUse.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  const handleCancel = () => {
+    setHasAcceptedTerms(false);
+    navigate(Pages.signUpWelcome());
+  };
+
+  return (
+    <Layout title="Terms of Service" isPublic>
+      <Stack gap="$8" align="center">
+        <Header
+          title="Terms of Service"
```
Comment (luizstacio) [inline packages/app/src/systems/SignUp/pages/TermsOfUse/TermsOfUse.tsx:34]:
```suggestion
          title="Terms of use Agreement"
```

**Review by luizstacio — APPROVED** [review]

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**